src/__tests__/utils/fixtures.ts

```diff
@@ -142,7 +142,7 @@ const getTestData = () => {
       is_improvement: true,
       is_regression: false,
       is_meaningful: true,
-      parent_signature: '4769486',
+      parent_signature: null,
```

Comment:
I think it makes more sense to leave them as `null` for now - when adding tests we can adjust.